third_party/move/move-compiler-v2/src/file_format_generator/module_generator.rs

```diff
@@ -1767,22 +1771,134 @@ impl ModuleContext<'_> {
                             no_args(name.as_str());
                             result.push(FF::FunctionAttribute::ModuleLock)
                         },
+                        well_known::PACK => {
+                            no_args(name.as_str());
+                            result.push(FF::FunctionAttribute::Pack)
+                        },
+                        well_known::UNPACK => {
+                            no_args(name.as_str());
+                            result.push(FF::FunctionAttribute::Unpack)
+                        },
                         _ => {
                             // skip
                         },
                     }
                 },
-                Attribute::Assign(_, name, _) => {
+                Attribute::Assign(_, name, attribute_value) => {
                     let name = fun_env.symbol_pool().string(*name);
                     if matches!(
                         name.as_str(),
-                        well_known::PERSISTENT_ATTRIBUTE | well_known::MODULE_LOCK_ATTRIBUTE
+                        well_known::PERSISTENT_ATTRIBUTE
+                            | well_known::MODULE_LOCK_ATTRIBUTE
+                            | well_known::PACK
+                            | well_known::UNPACK
```

Comment:
I don't understand why we exclude test variant here, but later check for it below.

Reply:
good catch, test variant is removed